app/components/team/index.js

```diff
@@ -0,0 +1,171 @@
+"use client"
+
+import { useState } from 'react';
+import { motion as m } from 'framer-motion'
+import Link from 'next/link'
+import './Team.scss'
+import './Teammate.scss'
+
+const copyMessageSuccess = 'Copied to clipboard'
+const copyMessageError = 'Failed to copy, sorry'
+
+function Teammate({name, role, gpgkey, email, imgSrc, links, className = ''}) {
+    const [copyMessage, setCopyMessage] = useState(copyMessageSuccess)
+    const [copyMessagesState, setCopyMessagesState] = useState({
+        email: false,
+        fieldId: false
+    });
+
+    function copyText(copytext, fieldId) {
+        try {
+            navigator.clipboard.writeText(copytext);
```

Comment:
I think we can move that in the component and reuse it to avoid code duplication. As an idea, since copy buttons looks almost identical, we can make an additional component that will take message to copy as a prop, and show copyMessage as a tip / popup.

Let's talk about that in private